### PR TITLE
spec: move non-shared schemas into respective specs

### LINKED ISSUE #239: Fix Grant Request `access_token` field in ReadMe
Instead of:
Screen Shot 2022-12-22 at 1 51 39 AM

it should show the proper options to specify the `access_token` object

### PR DESCRIPTION
**Pull request by sabineschaller**
## Changes proposed in this pull request

We had a bunch of schemas in the schemas.yaml file which were not shared anymore because we move the grant interaction and token introspection schemas out of this repo. I only kept the schemas shared between RS and AS in the schemas.yaml file.

## Context

- fixes #239

### CODE REVIEW

**Comment by mkurapov** [comment]
Can you push to `integration` so we can double check docs look good on https://open-payments-integration.readme.io/?

**Comment by sabineschaller** [comment]
> Can you push to `integration` so we can double check docs look good on https://open-payments-integration.readme.io/?

Uh, I didn't know we had that. I pushed it and I think it looks good @mkurapov

File: openapi/auth-server.yaml [hunk 1]
```diff
@@ -362,51 +500,43 @@ components:
...
+      type: string
+      description: '[ISO8601 repeating interval](https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals)'
+      examples:
+        - 'R11/2022-08-24T14:15:22Z/P1M'
+        - 'R/2017-03-01T13:00:00Z/2018-05-11T15:30:00Z'
+        - 'R-1/P1Y2M10DT2H30M/2022-05-11T15:30:00Z'
+    limits-outgoing:
+      title: limits-outgoing
+      description: Open Payments specific property that defines the limits under which outgoing payments can be created.
+      type: [object, 'null']
```
Comment (mkurapov) [inline openapi/auth-server.yaml:514]:
```suggestion
```
Do we need it if its non-required already?
also:
Screenshot 2023-03-10 at 13 55 04

File: openapi/auth-server.yaml [hunk 2]
```diff
@@ -362,51 +500,43 @@ components:
...
+              - interval
+        - required:
+            - sendAmount
+        - required:
+            - receiveAmount
+    list-actions:
+      enum:
+        - list
+        - list-all
+    read-actions:
```
Comment (mkurapov) [inline openapi/auth-server.yaml:536]: Do we need `list-actions` & `read-actions` if they aren't used?

**Review by mkurapov — COMMENTED** [review]
Cool to see the `access` working!
Screenshot 2023-03-10 at 13 49 59

File: openapi/auth-server.yaml (hunk 1 above)
Comment (sabineschaller) [reply openapi/auth-server.yaml:514]: If it's non-required, I think we can just put `type: object`

File: openapi/auth-server.yaml (hunk 2 above)
Comment (sabineschaller) [reply openapi/auth-server.yaml:536]: > Do we need `list-actions` & `read-actions` if they aren't used?

What do you mean?

Comment (mkurapov) [reply openapi/auth-server.yaml:536]: L523 `list-action` & L536 `read-actions` schemas aren't referenced anywhere in the repo, hence the GH Actions warning

Comment (sabineschaller) [reply openapi/auth-server.yaml:536]: Oh, I didn't realize that. I'll investigate

**Review by mkurapov — APPROVED** [review]

**Pull request merged**